app/views.py

```diff
@@ -702,4 +711,10 @@ def result(quizID=None):
     if quizID:
         quiz = validate_quiz_stu(quizID)
     correct, questions = get_response_answer(current_user, quizID)
-    return render_template('result.html', questions=questions, correct=correct, quiz=quiz)
\ No newline at end of file
+    return render_template('result.html', questions=questions, correct=correct, quiz=quiz)
+
+@app.route('/class/<int:groupID>/classquiz')
```

Comment:
```suggestion
@app.route('/class/<int:groupID>/quizzes')
```
Class is already at the front.